src/test/resources/com/znsio/teswiz/features/vodqa.feature

```diff
@@ -0,0 +1,10 @@
+@vodqa
+
+Feature: Vodqa test
+
+  @android
+  Scenario: Scrolling using fromPoint and toPoint
```

Comment:
Scrolling using - looks very odd.

it should be like - Validating scroll functionality in the App